docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: "3.9"
+
```

Comment:
[The version flag has been removed.](https://docs.docker.com/reference/compose-file/legacy-versions/)

```suggestion
```